### ISSUE TITLE
Missing: no documentation exists for CLI task to manage plugins

### ISSUE THREAD

**Issue by fiver-watson** [artefactual/atom-docs#218]
**Relevant version**

2.6, 2.7, ???? (not yet sure how long this task has existed)

**Relevant page**

Primary page: 

* https://www.accesstomemory.org/docs/2.7/admin-manual/maintenance/cli-tools/

Pages that deserve either a brief section, or at minimum a mention and a link: 

* https://www.accesstomemory.org/en/docs/2.7/admin-manual/maintenance/troubleshooting/#maintenance-troubleshooting
  * Adding something to the FAQ section would be really helpful for users searching the docs
* https://www.accesstomemory.org/docs/2.7/admin-manual/customization/authentication/#enabling-cas-authentication
  * Adding an additional tip, or adding to the existing TIP admonition, would be very helpful. Could also be just generally added on this page, since users could also lock themselves out accidentally enabling LDAP
* https://www.accesstomemory.org/en/docs/2.7/user-manual/administer/manage-plugins/#manage-plugins
  * See also admonition somewhere on this page would be helpful

**Problem encountered**

TL;DR: A potentially very useful command-line task is currently not documented. 

Context: 

Learned of this task recently when trying to help a user who had accidentally enabled the CAS authentication plugin, and therefore could not login without disabling it again. The same issue could potentially happen with LDAP. Thought that SQL would be necessary to disable the setting... but it turns out there is an undocumented CLI task precisely for this use case???

The file: 

* https://github.com/artefactual/atom/blob/qa/2.x/lib/task/tools/atomPluginsTask.class.php

The related forum thread: 

* https://groups.google.com/g/ica-atom-users/c/lOeJjEbqHbo/m/cmxV1zv9AgAJ

**Suggested fix**

This task should be added to the CLI tasks documentation page, and mentions/links should be added to a number of related pages for easier discovery. 

-----

Basic syntax of the task, for reference until documented

```
php symfony tools:atom-plugins [option] [pluginName]
```

Examples: 

List all plugins:

```
php symfony tools:atom-plugins list
```

Disable a plugin

```
php symfony tools:atom-plugins delete [pluginName]
```

Enable a plugin

```
php symfony tools:atom-plugins add [pluginName]
```
